spec/lib/efile/md/md502_calculator_spec.rb

```diff
@@ -3,7 +3,8 @@
 describe Efile::Md::Md502Calculator do
   let(:filing_status) { "single" }
   let(:county) { "Allegany" }
-  let(:intake) { create(:state_file_md_intake, filing_status: filing_status, residence_county: county) }
+  let(:includes_spouse) { filing_status == "married_filing_jointly" ? :with_spouse : nil}
```

Comment:
this felt like a good-ish way of when the filing status is set to "married_filing_jointly" adding the `:with_spouse` trait.

Reply:
cool! ty for figuring this out